### DISCUSSION TITLE
Allow to get the exact date and time before sending a Grafana Alert in the contact points, such as Telegram

### DISCUSSION THREAD

**Discussion by Carolina0802** [grafana/grafana#57011] [Alerting]
Allow to get the exact date and time before sending a Grafana Alert in the contact points, such as Telegram.

Currently, within the information that can be passed to the template data, there is the StartsAt parameter in which information can be passed, but about the time when the alert started to fire and not when the error occurred.

**Comment by eidermauricio** [comment]
It is a good idea to allow passing more variables like the ones of the query in the alert rule creation. So that in the message of the alert one can make more consistent notifications I mean with more contextual data.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
